RELEASES.md

```diff
@@ -0,0 +1,13 @@
+# 0.13.0
+
+- adopt the new `Durability` API proposed in [RFC #6]
+    - this replaces and generalizes the existing concepts of constants
+- remove "volatile" queries -- instead, you can use
+  `report_untracked_read` method on the salsa runtime
+- introduce "slots", an optimization to salsa's internal workings
+- document `#[salsa::requires]` attribute, which permits private dependencies
+- Adopt `AtomicU64` for `runtimeId` (#182)
+- use `ptr::eq` and `ptr::hash` for readability
+- upgrade parking lot, rand dependencies
```

Comment:
https://github.com/salsa-rs/salsa/pull/174 should be mentioned, it's the breaking change in this release

Reply:
I thought I did...

Reply:
I guess I could highlight it more explicitly